src/ray/raylet/scheduling/scheduling_policy.cc

```diff
@@ -204,5 +204,37 @@ int64_t SchedulingPolicy::HybridPolicy(const ResourceRequest &resource_request,
                                 require_available, is_node_available);
 }
 
+int64_t SchedulingPolicy::RandomPolicy(const ResourceRequest &resource_request,
+                                       std::function<bool(int64_t)> is_node_available) {
+  int64_t best_node = -1;
+  if (nodes_.empty()) {
+    return best_node;
```

Comment:
```suggestion
    return -1;
```
Very minor (just a style preference)